roles/cluster/cluster_deploy_kepler/tasks/main.yml

```
@@ -0,0 +1,23 @@
+- name: Grab kepler-operator source
+  command:
+    git clone https://github.com/sustainable-computing-io/kepler-operator "{{ artifact_extra_logs_dir }}/kepler"
```

```suggestion
    git clone https://github.com/sustainable-computing-io/kepler-operator --quiet "/tmp/kepler"
```
`--quiet` will make the logs more readable
`"/tmp/kepler"` is to avoid cluttering the artifact dir with unnecessary files

maybe you can add this command:
```
git --git-dir "/tmp/kepler/.git" show
```
to track which commit was deployed

---

I added showing the git commit that we are using, but using `--quiet` caused more complications than it's probably worth. Looks like `--quiet` puts the process in the background as well, surprisingly different than `wget`, so its more difficult to track when cloning is done.